multiversx_sdk_cli/tests/testdata/TestUser.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": 4,
```

Comment:
name the files `testUser*` (lowercase) ?

Reply:
Renamed